src/generators.ts

```diff
@@ -83,7 +83,16 @@ export class FolderGenerator<T> implements IGenerator<T> {
     innterContext.workingDirectory = path.join(context.workingDirectory, folderName);
     innterContext = resolveContextModifier(innterContext, this._contextMap);
 
-    await fs.mkdir(innterContext.workingDirectory);
+    if((context.options as any).overWriteDir === true) {

```

Comment:
* please, use `innterContext` instead of `context`
* to avoid `(context.options as any)` add `overWriteDir` to type definition
* suggest use `TemplateOptions.overWriteDir.false` instead of `true` like in `cli.ts`

Reply:
Updated. Please have a look again.